images/fcos-bastion-image/Containerfile

```diff
@@ -2,5 +2,5 @@ FROM quay.io/openshifttest/fcos-bm:base
 
 COPY root/ /
 
-RUN set -x; pip3 install --target=/usr/lib redfish pandas \
+RUN set -x; pip3 install --target=/usr/lib redfish pandas netaddr wheel git+https://github.com/jadhaj/py-junos-eznc.git@fix \
```

Comment:
mmm looking at your fix there, it seems we should wait for https://github.com/Juniper/py-junos-eznc/pull/1281/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552 to have support for python4.12.

Reply:
yes. looks like

Reply:
I'm unsure, but to unblock our job, we could try installing from either

https://github.com/Juniper/py-junos-eznc/pull/1281

or https://github.com/Juniper/py-junos-eznc/pull/1288

by pulling those branches into your personal fork temporarily.

Reply:
PR https://github.com/Juniper/py-junos-eznc/pull/1281 was merged and closed